packages/jellyfish-api-core/__tests__/category/loan/getLoanInfo.test.ts

```diff
@@ -0,0 +1,191 @@
+            }
+          },
+          'Loan tokens': {},
+          'Loan schemes': [],
+          collateralValueUSD: new BigNumber(0),
+          loanValueUSD: new BigNumber(0)
+        }
+      )
+    }
+  })
+
+  it('should getLoanInfo - include loan schemes', async () => {
+    // Before createLoanScheme
+    {
+      const data = await testing.rpc.loan.getLoanInfo()
+      expect(data).toStrictEqual(
+        {
+          'Collateral tokens': {},
+          'Loan tokens': {},
+          'Loan schemes': [],
+          collateralValueUSD: new BigNumber(0),
+          loanValueUSD: new BigNumber(0)
+        }
+      )
+    }
+
+    await testing.container.call('createloanscheme', [100, new BigNumber(1.5), 'default'])
+    await testing.container.generate(1)
+
+    await testing.container.call('createloanscheme', [200, new BigNumber(2.5), 'scheme'])
+    await testing.container.generate(1)
+
+    // After createLoanScheme
+    {
+      const data = await testing.rpc.loan.getLoanInfo()
+      expect(data).toStrictEqual(
+        {
+          'Collateral tokens': {},
+          'Loan tokens': {},
+          'Loan schemes': [{
+            id: 'default',
+            mincolratio: new BigNumber(100),
+            interestrate: new BigNumber(1.5),
+            default: true
+          }, {
+            id: 'scheme',
+            mincolratio: new BigNumber(200),
+            interestrate: new BigNumber(2.5),
+            default: false
+          }],
+          collateralValueUSD: new BigNumber(0),
+          loanValueUSD: new BigNumber(0)
+        }
+      )
+    }
+  })
+
+  it('should getLoanInfo - include loan tokens', async () => {
+    // Before setLoanToken
+    {
+      const data = await testing.rpc.loan.getLoanInfo()
+      expect(data).toStrictEqual(
+        {
+          'Collateral tokens': {},
+          'Loan tokens': {},
+          'Loan schemes': [],
+          collateralValueUSD: new BigNumber(0),
+          loanValueUSD: new BigNumber(0)
+        }
+      )
+    }
+
+    // setLoanToken with new token
+    const anotherTokenOracleId = await testing.container.call('appointoracle', [await testing.generateAddress(), [{
+      token: 'APPL',
+      currency: 'USD'
+    }], 1])
+    await testing.generate(1)
+    const loanTokenId = await testing.container.call('setloantoken', [{
+      symbol: 'APPL',
+      name: 'APPLE',
+      priceFeedId: anotherTokenOracleId,
+      interest: new BigNumber(0.01)
+    }])
+    await testing.generate(1)
+    const height = new BigNumber(await testing.container.getBlockCount())
+
+    // After setLoanToken
+    {
+      const data = await testing.rpc.loan.getLoanInfo()
+      expect(data).toStrictEqual(
+        {
+          'Collateral tokens': {},
+          'Loan tokens': {
+            [loanTokenId]: {
+              token: {
+                2: {
+                  collateralAddress: GenesisKeys[0].owner.address,
+                  creationHeight: new BigNumber(height),
```

Comment:
Height is already a BigNumber.  You may change it to `creationHeight: height,`

Reply:
`creationHeight: new BigNumber(await testing.container.getBlockCount()),` even better